docs/API.md

```diff
@@ -477,7 +477,7 @@ minioClient.fGetObject('mybucket', 'photo.jpg', '/tmp/photo.jpg', function(err)
 
 ```
 <a name="putObject"></a>
-### putObject(bucketName, objectName, stream, size, contentType, callback)
+### putObject(bucketName, objectName, stream, size, callback)
```

Comment:
contentType is still needed when a user wishes to provide contentType . IT is a necessary input parameter.